bin/commands/internal/utils/cleanup-ipcmq/index.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+_CEE_RUNOPTS="XPLINK(ON),HEAPPOOLS(OFF),HEAPPOOLS64(OFF)" ${ZWE_zowe_runtimeDirectory}/bin/utils/configmgr -script "${ZWE_zowe_runtimeDirectory}/bin/commands/internal/utils/cleanup-ipcmq/cli.js"
```

Comment:
[HEAPPOOLS/64](https://github.com/zowe/zowe-common-c/pull/497) could be omitted.